src/index.html

```diff
@@ -11,6 +11,47 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="/action_page.php"
+      method="post"
+      data-qa="big"
+      class="form"
+    >
+      <label>
+        <img
+          src="images/Search.svg"
+          alt="searchbar"
+          class="image"
+        >
+        <input
+          type="search"
+          class="search"
+          name="search"
+          placeholder="Try &quot;Los Angeles&quot;"
+          data-qa="keypress"
+        >
```

Comment:
```suggestion
        <img
          src="images/Search.svg"
          alt="searchbar"
          class="image"
        >
        
        <input
          type="search"
          class="search"
          name="search"
          placeholder="Try &quot;Los Angeles&quot;"
          data-qa="keypress"
        >
```

Reply:
is not fixed. add empty line between <img> and <input> as previous mentor asked